.rubocop.yml

```diff
@@ -7,6 +7,7 @@ AllCops:
     - "Rakefile"
     - "*.gemspec"
     - "config.ru"
+    - "lib/browser/version.rb"
```

Comment:
rubocop notice `Style/RedundantFreeze`. but it should be freezed.

https://github.com/fnando/browser/blob/5f13125e6b5f9d86edbc1465591cbd081a8937d8/lib/browser/version.rb#L3-L5